### PR TITLE
43/load balances effect

### PR DESCRIPTION
**Pull request by anxolin**
**Includes**
- [X] Creates an effect to fetch the balances, it uses the `tokenApi` and `depositApi`
- [X] The component will handle the display of the balances, the display of an error in case the fetching has any errors, and the display of a loading message while the balances are being loaded

**Doesn't include**
- Style for Loading or Error, this will be addresses in a separate PR

Part of #43

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr53--dexreact.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr53--dexreact.review.gnosisdev.com

File: src/effects/useTokenBalances.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import { tokenApi } from 'api'
+
+interface UseTokenBalanceResult {
+  balances: TokenBalanceDetails[] | undefined
+  error: boolean
+}
+
+async function _getBalances(): Promise<TokenBalanceDetails[]> {
+  const tokens = tokenApi.getTokens(Network.Rinkeby)
+  const userAddress = '0x1111111111111111111111111111111111111111'
```
Comment (alfetopito) [inline src/effects/useTokenBalances.ts]: Just to be clear, this is a temporary value, we'll later get this value as input from the connected wallet, correct?

File: src/effects/useTokenBalances.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+  const balancePromises = tokens.map(async (token, index) => {
+    const tokenAddress = token.address
+    const [exchangeWallet, pendingDeposits, pendingWithdraws] = await Promise.all([
+      depositApi.getBalance(userAddress, tokenAddress),
+      depositApi.getPendingWithdrawAmount(userAddress, tokenAddress),
```
Comment (alfetopito) [inline src/effects/useTokenBalances.ts]: `getPendingWithdrawAmount` is repeated. Did you mean `getPendingDepositAmount`?

File: src/effects/useTokenBalances.ts (hunk 1 above)
Comment (anxolin) [reply src/effects/useTokenBalances.ts]: You'll see it in one of the standing PRs :)

File: src/effects/useTokenBalances.ts (hunk 2 above)
Comment (anxolin) [reply src/effects/useTokenBalances.ts]: totally, good catch

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr53--dexreact.review.gnosisdev.com

**Pull request merged**